c/src/point_one/polaris/polaris.h

```diff
@@ -76,9 +76,13 @@
 #define POLARIS_CONNECTION_CLOSED -7
 #define POLARIS_TIMED_OUT -8
 
-typedef void (*PolarisCallback_t)(const uint8_t* buffer, size_t size_bytes);
+struct PolarisContext_s;
+typedef struct PolarisContext_s PolarisContext_t;
```

Comment:
I'm not sure this will actually work with a C99 compiler

Reply:
Doing `typedef struct Foo_s Foo_t`, the forward declare, or the function pointer using the type? In either case, I just tested them with `-std=c99` and they all work fine.

Reply:
I was actually referring to the struct definition below. It's c++ style but I suppose if it works it works!